web/source/kmwosk.ts

```diff
@@ -1322,7 +1321,7 @@ if(!window['keyman']['initialized']) {
      **/
     osk.showLanguageMenu = function()
     {
-      var n=0,kbdList=keymanweb._KeyboardStubs,nKbds=kbdList.length;
+      var n=0,kbdList=(<KeymanBase>keymanweb).keyboardManager.keyboardStubs,nKbds=kbdList.length;
```

Comment:
Do we need this cast?

Reply:
Not at all; it was for similar reasons to the other noted cast.